ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-main.yaml

```diff
@@ -57,42 +47,14 @@ resources:
     requests:
       cpu: "16"
       memory: 24Gi
-  vllm-build-main-rocm:
-    limits:
-      cpu: "16"
-      memory: 35Gi
-    requests:
-      cpu: "16"
-      memory: 35Gi
 tests:
-- always_run: true
-  as: smoke-test
-  cluster: build05
-  commands: |
-    export VIRTUAL_ENV=/opt/vllm
-    export PATH=$VIRTUAL_ENV/bin/:$PATH
-
-    cd extras
-
-    bash smoke-test.sh
-  container:
-    clone: true
-    from: vllm-build-main-cuda
```

Comment:
I'd rather not delete the smoke test definition, this definition is working, the test is failing because the pod fails to schedule, and we're working to fix that.

Reply:
We cannot afford not to have a smoke test. See https://github.com/openshift/release/pull/56617 as well as the linked discussion.